internal/resources/cluster/integration/resource_integration.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package integration
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/pkg/errors"
+
+	clienterrors "github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/errors"
+	integrationclient "github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/integration"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/cluster/integration"
+)
+
+func ResourceIntegration(alternateClients ...integrationclient.ClientService) *schema.Resource {
+	ri := &resourceIntegration{alternateClients: append(alternateClients, &defaultClient{})}
+
+	return &schema.Resource{
+		CreateContext: ri.resourceIntegrationCreate,
```

Comment:
What about the updateContext ?

Reply:
the TMC API doesn't support Update - maybe you can suggest how to handle delete/create instead of update when the API doesn't support it?